gateway/src/test/java/io/zeebe/mocks/RequestFutureMock.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright © 2017 camunda services GmbH (info@camunda.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.zeebe.mocks;
+
+import io.zeebe.broker.client.api.ZeebeFuture;
+import io.zeebe.broker.client.api.commands.Topology;
+import io.zeebe.broker.client.api.commands.TopologyRequestStep1;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.TimeoutException;
+
+public class RequestFutureMock<E> implements ZeebeFuture<E>, TopologyRequestStep1 {
+
+  private E self;
+
+  @Override
+  public E join() {
+    return null;
+  }
+
+  @Override
+  public E join(final long timeout, final TimeUnit unit) {
+    return null;
+  }
+
+  @Override
+  @SuppressWarnings("unchecked")
+  public ZeebeFuture<Topology> send() {
+    return (ZeebeFuture<Topology>) this;
+  }
+
+  @Override
+  public boolean cancel(final boolean mayInterruptIfRunning) {
+    return false;
+  }
+
+  @Override
+  public boolean isCancelled() {
+    return false;
+  }
+
+  @Override
+  public boolean isDone() {
```

Comment:
Should return `true`

Reply:
Yes, you are correct. But generated code should not be reviewed and since we discussed that this is gonna be revisited I see no value in changing this now.